source/includes/documentation/_vue_js_elements.md

```diff
@@ -1548,47 +1548,243 @@ and [JDI vuetify page tests for windows](https://github.com/jdi-testing/jdi-ligh
+    public static void hasDividerLinesListTest() {
+      threeLineList.show();
+      threeLineList.divider(1).is().horizontal();
+    }
+
+    //@FindBy(css = "#ThreeLineList .v-list")
+    @UI("#ThreeLineList .v-list")
+    public static VuetifyList threeLineList;
+    
+    @Test(description = "Test checks that list items has avatars")
+    public static void hasAvatarLinesListTest() {
+      threeLineList.show();
+      threeLineList.item(1).avatar().is().displayed();
+    }
+    
+    //@FindBy(css = "#ActionAndItemGroupsList .v-list[2]")
+    @UI("#ActionAndItemGroupsList .v-list[2]")
+    public static VuetifyList actionAndItemGroupsGeneralList;
+    
+    @Test(description = "Test shows how to test lists with checkboxes")
+    public static void checkboxListTest() {
+      ListItem item = actionAndItemGroupsGeneralList.item(1);
+      VueCheckbox itemCheckbox = item.checkbox();
+      item.show();
+      item.is().clickable().and().notActive();
+      itemCheckbox.is().displayed().and().enabled().and().unchecked();
+      itemCheckbox.check();
+      itemCheckbox.is().checked();
+      item.is().active();
+      item.click();
+      item.is().notActive();
+      itemCheckbox.is().unchecked();
+    }
+
+    //@FindBy(css = "#ActionStackList .v-list")
+    @UI("#ActionStackList .v-list")
+    public static VuetifyList actionStackList;  
+      
+    @Test(description = "Test checks list's icon types")
+    public static void iconsListTest() {
+      ListItem item = actionStackList.item(1);
+      item.show();
+      item.actionText().has().text("15 min");
+      item.is().clickable();
+      item.click();
+      item.is().active();
+      item.icon().has().type("mdi-star");
+      item.click();
+      item.is().notActive();
+      item.icon().has().type("mdi-star-outline");
+    }
+    
+    @UI("#DarkList .v-list")
+    public static VuetifyList darkList;
+    @UI("#ElevatedList .v-list")
+    public static List<VuetifyList> elevatedLists;
+    
+    @Test(description = "Test checks list's theme")
+    public void themeListsTest() {
+      darkList.show();
+      darkList.has().darkTheme();
+      elevatedLists.get(1).show();
+      elevatedLists.get(1).has().lightTheme();
+    }
+
+    @UI("#DisabledList .v-list")
+    public static VuetifyList disabledList;
+    @UI("#DenseList .v-list")
+    public static VuetifyList denseList;
+
+    @Test(description = "Test checks if list is dense or not")
+    public void denseListsTest() {
+      denseList.show();
+      denseList.is().dense();
+      disabledList.show();
+      disabledList.is().notDense();
+    }
+
+    @UI("#DarkList .v-list")
+    public static VuetifyList darkList;
+    @UI("#FlatList .v-list")
+    public static VuetifyList flatList;
+    
+    @Test(description = "Test checks if list is flat or not")
+    public void flatListsTest(){
+        flatList.show();
+        flatList.is().flat();
+        darkList.show();
+        darkList.is().notFlat();
+    }
```

Comment:
If I am not mistaken we discussed that we take 1-2-3 tests, not all of them :)
And also master task says '2.	Element usage example (typically we use **one** of element's tests for that)'

Reply:
Done. Only two tests examples are left now.